src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -25,6 +25,44 @@ export function trimPackage(pkg) {
   return filename.replace(/\.(dylib|so|a|dll|exe)$/, '');
 }
 
+class FunctionName extends React.Component {
+  static propTypes = {
+    frame: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      fullFunction: false,
+    };
+  }
+
+  toggle(event) {
```

Comment:
```suggestion
  toggle = (event) => {
```

Using a fat arrow function will let you skip the `.bind(this)` in the render() call.